dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -31,6 +32,9 @@
     /** Map from AST {@link Tree}s to {@link Node}s. */
     protected final IdentityHashMap<Tree, Node> treeLookup;
 
+    /** Map from AST {@link UnaryTree}s to compound {@link AssignmentNode}s. */
+    protected final IdentityHashMap<UnaryTree, AssignmentNode> compoundAssignTreeLookupMap;
```

Comment:
Because `treeLookup` and `convertedTreeLookup` are reserved for a node for unary operation 
self and a conversion node like unbox or widening or something like that.

Please consider following code:
```
@IntVal(0)
short n = 0;
@IntVal(0)
int m = n++;
```

At `visitUnary` for `n++`, artificial tree and node for assignment(`n = n + 1`) will be created.
And `temp = n`(where `n` is a node of before the assignment) will be created and stored in `treeLookup` because it's postfix.

At `visitAssignment` for `int m = n++`, `WideningConversionNode` for `temp = n` will be created and stored into `convertedTreeLookup`.

We need `n = n + 1` to type check in `BaseTypeVisitor`, I need to store it into `unaryAssignNodeLookupMap`